### ISSUE TITLE
Update Template with the box for Checking Memory Leaks

### ISSUE THREAD

**Issue by danston** [CGAL/cgal#5817]
As discussed in [this](https://github.com/CGAL/cgal/pull/5737#discussion_r660575086) comment to the PR with the new Weights package, the feature submission template should be updated by adding a box to tick that the leak check has been done. This issue is a reminder.

**Comment by lrineau** [comment]
The wiki element that must be modified is the Mediawiki template https://cgal.geometryfactory.com/CGAL/Members/w/index.php?title=Template:Feature

It is a complicated template, that interacts with the property system of [Semantic Mediawiki](https://cgal.geometryfactory.com/CGAL/Members/w/index.php?title=Template:Feature).

**Comment by afabri** [comment]
You mean it is more complicated than just adding a line to an ascii file?

**Comment by lrineau** [comment]
Yes, indeed. Here is the source code: https://cgal.geometryfactory.com/CGAL/Members/w/index.php?title=Template:Feature&action=edit

I think it would be far easier to add such a think in the *Github* template for pull-requests.

**Comment by afabri** [comment]
I agree. Maybe you or Maxime can just do it.

**Comment by lrineau** [comment]
The quick and dirty way is to add something in the current template, here: https://github.com/CGAL/cgal/edit/master/.github/PULL_REQUEST_TEMPLATE.md.

A proper way would be to pass to a system with multiple PR templates:
  - for features,
  - for small features,
  - and a third for bug-fixes and undocumented contributions.
